mautic/mautic
# Fix list segments

Please answer the following questions:

| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Fixed issues  |  no

## Description
- This PR fixes the tool tip for list view in contacts
- Changes list icon to pie chart icon for batch edit segments

## Steps to test this PR
- apply PR
- click the contacts menu
- tool tip for icon to change to list view should now read list view instead of segment view
- list icon for batch edit segments should now be a pie chart icon
